pkg/metering/units.go

```diff
@@ -0,0 +1,20 @@
+package metering
+
+var (
+	PayloadMeteringUnit = unit{Name: "payload", Unit: "bytes"}
+
+	// ComputeMeteringUnit is an example. The compute cap will eventually be obsoleted
+	// by the CRE No-DAG SDK.
+	ComputeMeteringUnit = unit{Name: "compute", Unit: "seconds"}
```

Comment:
I think we can drop `metering` from var names too for the same reason:
```suggestion
	PayloadUnit = unit{Name: "payload", Unit: "bytes"}

	// ComputeUnit is an example. The compute cap will eventually be obsoleted
	// by the CRE No-DAG SDK.
	ComputeUnit = unit{Name: "compute", Unit: "seconds"}
```

Reply:
Although, either way, we have redundant suffix and field: `Unit.Unit`. Is there a better name for one of them?

Reply:
> I think we can drop metering from var names too for the same reason

Agreed, 78bed2a17c5a40e7fcefb9eb918ee4470ee022da

> Is there a better name for one of them?

`unit` --> `unitType`? `meteredType`? `meteredUnit`? `unitAndName`? 

The struct name `unit` is unexported so its trivial to update later if we can't think of something now, and is the better candidate for renaming since the inner `Unit` field is exactly what the name suggests and nothing more.